README.md

```diff
@@ -119,6 +119,15 @@ The "job" element will have the following elements, but is not restricted to the
 }
 ```
 
+Note: Configuring jobs to run with `latest` composer dependencies is not endorsed.
+      The default generated matrix will contain entries for `lowest` and `locked` dependencies.
+      This action operates under the assumption that you do your due diligence, and keep `composer.json`
+      and `composer.lock` updated.
+      By operating on locked dependencies, you will be able to pinpoint the exact dependency
+      change that caused a test regression.
+      We endorse regularly updating dependencies with automation like [renovate](https://github.com/renovatebot) or
+      [dependabot](https://github.com/dependabot).
```

Comment:
what renovate/dependabot config entry would be required to have these updates?

besides this, it seems that lockfile updates do not properly handle latest. So we might want to have an additional action to be registered - or we go the way that we do only run latest for auto-contributions such as dependabot/renovate.

Reply:
The lockfile maintenance is as if you run composer update on the PHP version specified in `config.platform.php`, so we won't get updates that require a higher minimum PHP version.